rules/falco_rules.yaml

```diff
@@ -2536,25 +2598,22 @@
 - rule: Launch Suspicious Network Tool in Container
   desc: Detect network tools launched inside container
   condition: >
-    spawned_process and container and network_tool_procs and not user_known_network_tool_activities
+    spawned_process 
+    and container 
+    and network_tool_procs 
+    and not user_known_network_tool_activities
   output: Network tool launched in container (evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
   priority: NOTICE
   tags: [maturity_sandbox, container, network, process, mitre_execution, T1059]
 
-# This rule is not enabled by default, as there are legitimate use
-# cases for these tools on hosts. If you want to enable it, modify the
-# following macro.
-- macro: consider_network_tools_on_host
```

Comment:
One requirement in style guide is that they need to evaluate to true, we missed this here in previous cleanups.